pom.xml

```diff
@@ -176,18 +215,33 @@
             </dependency>
             <!-- == End Database == -->
 
+            <!-- == Begin Test == -->
+            <dependency>
+                <groupId>org.seleniumhq.selenium</groupId>
+                <artifactId>selenium-java</artifactId>
+                <version>${selenium.version}</version>
+            </dependency>
+            <!-- == End Test == -->
+
         </dependencies>
     </dependencyManagement>
     <properties>
+        <!-- == Maven Plugins == -->
+        <org.codehaus.mojo.build-helper-maven-plugi.version>1.9.1</org.codehaus.mojo.build-helper-maven-plugi.version>
+        <maven-war-plugin.version>2.5</maven-war-plugin.version>
+        <formatter-maven-plugin.version>0.5.2</formatter-maven-plugin.version>
+        <com.google.code.maven-license-plugin.version>1.4.0</com.google.code.maven-license-plugin.version>
+        <!-- == Other Properties== -->
         <encoding>UTF-8</encoding>
         <java-version>1.8</java-version>
-        <org.codehaus.mojo.build-helper-maven-plugi.version>1.9.1</org.codehaus.mojo.build-helper-maven-plugi.version>
         <postgresql.version>9.4-1206-jdbc41</postgresql.version>
         <jasperreports.version>3.5.1</jasperreports.version>
+        <selenium.version>2.46.0</selenium.version>
+        <project.root.basedir>${project.basedir}</project.root.basedir>
```

Comment:
@bttanakagn 
Let's that way. Creating issue or Reopening issue, either is fine. Thanks.